deploy-as-code/helm/environments/mgramseva-qa-secrets.yaml

```diff
@@ -5,6 +5,9 @@ cluster-configs:
             password: ENC[AES256_GCM,data:vPC6D29TkTcwtw66g/FSaw==,iv:CcTl/3Ez01EbaiQZitlb4hwhVmqmjP8nNH0jWnF163k=,tag:OV/czZSWCauZmWQ38Ip50A==,type:str]
             flywayUsername: ENC[AES256_GCM,data:i7BiRT7z,iv:oar3tezcuh0hLw7moc8eMxTW9v41dkb5AJxxybYd50o=,tag:BG9DbFV4UTxP+f4uG4yXhw==,type:str]
             flywayPassword: ENC[AES256_GCM,data:W2OxvhXREj4EYp+bl7wb/A==,iv:cSU57AgKr9kaW8/zw0FH7y+DCbn9FFkthy0WLLosp68=,tag:hoNO09iWmdrPwszc4iPdig==,type:str]
+            mongo-db-username: ENC[AES256_GCM,data:jCYkLFcbNZwy9A==,iv:QKp+PAh6snZ8gDJZ1KhdLb210PsL0N1dTdzJUCkbl0s=,tag:p4cKNgJuavzLNu6giQmtRA==,type:str]
```

Comment:
The secrets can't be copied by simple copy-paste. They have signature associated with them(present at the end of the file). You need to request DevOps team to copy them.